src/github.com/matrix-org/dendrite/roomserver/query/query.go

```diff
@@ -40,25 +40,20 @@ type RoomserverQueryAPIDatabase interface {
 	// Lookup the numeric IDs for a list of events.
 	// Returns an error if there was a problem talking to the database.
 	EventNIDs(eventIDs []string) (map[string]types.EventNID, error)
-	// Save a given room alias with the room ID it refers to.
```

Comment:
Yeah, looks like it was left over from an aborted attempt to fit the room alias API into the query API in the roomserver.